dishes/colombian/otras_preparaciones/cayeye.md

```
@@ -1,105 +1,29 @@
 ---
 
-## 📚 Fuentes y Enlaces
+## Ingredientes
 
-- [Receta de cayeye - Recetas de Colombia](https://www.recetasdecolombia.com/cayeye)
-- [Cayeye colombiano - YouTube](https://www.youtube.com/results?search_query=cayeye+colombiano)
-- [TikTok #cayeye](https://www.tiktok.com/tag/cayeye)
-
----
+*   6-8 guineos verdes (bananos verdes pequeños)
+*   4 cucharadas de mantequilla
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)